src/lua/fileffi.lua

```diff
@@ -339,6 +345,40 @@ local function createFileWrapper(wrapper)
     return file
 end
 
+local function createUvListener(port, cb)
+    if type(port) ~= 'number' then error 'port argument must be an unsigned number' end
+    if (port) == nil then error 'must provide a port' end
+    if cb == nil then error 'must provide a callback function' end
+
+    _callback = function(fifo)  end
+
+    if cb ~= nil then
```

Comment:
If we check for `cb == nil` before, this check isn't really useful.

Also, we're going to overwrite `_callback` afterward. And it should be declared as `local`.